
package com.laegler.openbanking.soap.service;

import javax.xml.ws.WebFault;


/**
 * Bad request
 *
 * This class was generated by Apache CXF 3.3.3
 * 2019-10-21T07:01:26.903+13:00
 * Generated source version: 3.3.3
 */

@WebFault(name = "PUT_ChangeEventSubscriptionsEventSubscriptionId_400", targetNamespace = "http://laegler.com/openbanking/soap/model")
public class PUTChangeEventSubscriptionsEventSubscriptionId400 extends Exception {

    private com.laegler.openbanking.soap.model.OBErrorResponse1 putChangeEventSubscriptionsEventSubscriptionId400;

    public PUTChangeEventSubscriptionsEventSubscriptionId400() {
        super();
    }

    public PUTChangeEventSubscriptionsEventSubscriptionId400(String message) {
        super(message);
    }

    public PUTChangeEventSubscriptionsEventSubscriptionId400(String message, java.lang.Throwable cause) {
        super(message, cause);
    }

    public PUTChangeEventSubscriptionsEventSubscriptionId400(String message, com.laegler.openbanking.soap.model.OBErrorResponse1 putChangeEventSubscriptionsEventSubscriptionId400) {
        super(message);
        this.putChangeEventSubscriptionsEventSubscriptionId400 = putChangeEventSubscriptionsEventSubscriptionId400;
    }

    public PUTChangeEventSubscriptionsEventSubscriptionId400(String message, com.laegler.openbanking.soap.model.OBErrorResponse1 putChangeEventSubscriptionsEventSubscriptionId400, java.lang.Throwable cause) {
        super(message, cause);
        this.putChangeEventSubscriptionsEventSubscriptionId400 = putChangeEventSubscriptionsEventSubscriptionId400;
    }

    public com.laegler.openbanking.soap.model.OBErrorResponse1 getFaultInfo() {
        return this.putChangeEventSubscriptionsEventSubscriptionId400;
    }
}
